ac_config.lua

```diff
@@ -7,6 +7,8 @@ Components = {
 	["client.superjump"] = false, -- protect against super jump cheats
 	["client.weaponblacklist"] = false, -- blacklist certain weapons in ac_c.lua
 	["client.carblacklist"] = false, -- blacklist certain cars in ac_c.lua
+	["client.misc"] = true, -- anti night, thermal vision, anti multi damage and spectate
+	["server.cleartask"] = true, -- detect kicking from car
```

Comment:
i would further elaborate on the comment, something like `detects if a player clears tasks of another player, e.g. kicks them from a car`